code/drasil-example/pdcontroller/lib/Drasil/PDController/Body.hs

```diff
@@ -104,6 +104,9 @@ si = SI {
   _authors = [naveen],
   _purpose = [],
   _background  = [],
+  _motivation  = [],
```

Comment:
Reviewing multiple examples in a row that all set these 3 things to be empty. This really does start to feel like a hack. If we're going to implement a feature, shouldn't we be using it in all the examples? Just using it in Projectile does not convince me that this is a feature that is implemented at the right level.

Reply:
We do have a form of this information that we write for the SRS documents, as we just recently did for #3777. It would be nice if we could share them with a chunk that defines what the purpose, background, and motivation of each case study, respectively, is.

This is also related to #3773. The specific purpose, background, and motivation of each case study are just instances of the generic concepts of purpose, background, and motivation. In other words, for our current design, these might just be unique `ConceptInstances` for each case study. Rather than having lists here, it might be that we just have a `UID` to the specific `ConceptInstance`.

Reply:
Well, actually, would it be nice to share the same SRS discussion of the purp., back., and moti. in the generated READMEs? I'm not sure anymore. We might want a longer version of that in the SRS, and a shorter one here. @smiths do you have any thoughts on this? Do you think we can share them?

Reply:
@balacij I think for every example we should be able (if we haven't already) to write a one sentence purpose.  This one sentence purpose would fit nicely in the README.  It should also be usable in the SRS. As @NoahCardoso recently did, we can reuse the purpose in the Introduction to the SRS.  Although we haven't done it yet, I could see us wanting to have a longer version of the purpose for the SRS.  For instance, our one sentence purpose could be something like: 

The software library provides a multi-domain Lattice Boltzmann solver.

(I'm working on a project with a colleague in mechanical engineering that could be described with that sentence.)  The problem with this one sentence purpose is that it introduces at least two concepts that not everyone will be familiar with.  This could be fine in a README file, but in the SRS, a description of what multi-domain means and what Lattice Boltzmann means would be in order.  Following #3773, we could say that the purpose is an idea definition that includes two other idea definitions.  In the README we might not generate the text for the definitions of the sub-ideas, but for the SRS, this would be a natural thing to do.

We could go through a similar conversation for Motivation and Scope.  I think Motivation would be fairly simple, but Scope does get more involved.  There are different views of Scope.  There is a simple "high level" view, and then there is the view that decomposes the Scope into Scope Decisions (things that are entirely left out) and Assumptions.  The Scope can also include typical values and examples.